### DISCUSSION TITLE
How to save object as monolithic file?

### DISCUSSION THREAD

**Discussion by lcs-crr** [danielhrisca/asammdf#767] [Q&A]
I've been trying to modify mf4 file to remove:
-channel names
-comments
-irrelevant channels/signals
When I then try to save the object as an mf4 file using _object.save(path)_ the saved file shows up as 18 fragments each around 5MB.
Is this behaviour intendend and if yes, is there a way to ensure the object is saved as a monolithic file?
Cheers.

**Comment by danielhrisca** [comment]
That is really strange. Maybe you are just calling save with `overwrite=False` so new files are created each time with a new name.

**Reply by lcs-crr** [reply]
It was a fault in my code, apologies.
Thank you for your help nevertheless.
